### DISCUSSION TITLE
Download a page as HTML does not work when there is an image on the page

### DISCUSSION THREAD

**Discussion by Karsten111** [outline/outline#6446] [Self Hosting]
Usually, all pages can be downloaded as HTML or as Markdown. Currently, this does not work when there is an image on the page.

**To Reproduce**
1. Select a page that contains text and at least one image.
2. Go to '...'
3. Click on 'Download'
4. Click on "HTML"
5. Nothing happens

**Expected behavior**
I expect a pop-up to open that will ask me how to open the HTML document. Instead, nothing happens.

**Outline (please complete the following information):**
- Install: self hosted
- Version: v0.74.0

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser Chrome
 - Version 120.0

**Comment by linear** [comment]
<p><a href="https://linear.app/outline/issue/OLN-213/download-a-page-as-html-does-not-work-when-there-is-an-image-on-the">OLN-213 Download a page as HTML does not work when there is an image on the page</a></p>

**Comment by tommoor** [comment]
Please provide server logs from the time of the request, most likely the server cannot access your image storage due to DNS issues – this is a fairly common issue with self-hosted installations.

**Reply by Karsten111** [reply]
Thanks for your answer. You are probably right. I will ask the Admin to check this and provide some server logs.

**Reply by Karsten111** [reply]
I confirm that the error was due to an issue on our server. I will close this bug.
